orchestrator/api/api_v1/endpoints/processes.py

```diff
@@ -232,20 +241,25 @@ def assignees() -> List[str]:
     return [assignee.value for assignee in Assignee]
 
 
-@router.get("/{pid}", response_model=ProcessSchema)
-def show(pid: UUID) -> Dict[str, Any]:
-    process = _get_process(pid)
+@deprecated("product (UUID) changed to product_id from version 1.2.3, removing after version 1.3.0")
+def convert_to_old_process(process: dict) -> dict:
+    return {**process, "product": process["product_id"]}
```

Comment:
Great work on making things backwards compatible and marking the deprecations!

I've created this ticket for the removal follow-up: https://github.com/workfloworchestrator/orchestrator-core/issues/342

Perhaps you can add some notes to it so that other people could pick it up as well. (or for yourself to remember :) )